setup.py

```diff
@@ -317,6 +320,7 @@ def get_long_description():
         packages=find_packages(),
         package_data={
             "": ["images/*", "*.ini"],
+            "envisage": ["examples/*"],
```

Comment:
Is this recursive? Could you check if the 'preferences.ini' in `plugins/tasks/attractors` (quite a few folders deep) file is included please?

Given `examples` is a package, you might want to split this out to make it more explicit what is data (vs packages, which get picked up by `find_packages` on line 320), e.g.:
```
"envisage.examples": [
    "Hello_World/*",
]
```
On the other hand, folders like "Hello_World" are now indistinguishable compared to packages such as `envisage.examples.tests`. It might be clearer to put things one level down, e.g. `envisage/examples/demo`.
Then you can do `"envisage.examples": ["demo/*", "demo/*/*", ...]`

Reply:
I went with the approach you described, but there are many nested folders so I ended up with:
```
"envisage.examples": [
                "demo/*",
                "demo/*/*",
                "demo/*/*/*",
                "demo/*/*/*/*",
                "demo/*/*/*/*/*",
                "demo/*/*/*/*/*/*",
            ],
```
in order to include everything

Reply:
Like a little christmas tree 🎄 . I don't know of a better solution apart from doing an `os.path.walk(... topdown=False)` once to determine how deep the file structure is.

Reply:
The christmas tree is probably good enough for now.